shell/packages/sandstorm-ui-grainlist/grainlist-client.js

```diff
@@ -80,11 +80,19 @@ Template.sandstormGrainList.helpers({
   filteredSortedGrains: filteredSortedGrains,
   searchText: function() {
     return Template.instance().data._filter.get();
+  },
+  myGrainsCount: function () {
+    return Template.instance().data._db.currentUserGrains({}, {}).fetch().length;
+  },
+  myGrainsSize: function () {
+    // TODO(cleanup): extract prettySize and other similar helpers from globals into a package
+    return prettySize(Template.instance().data._db.getMyUsage().storage);
```

Comment:
Please use `Meteor.user().storageUsage`, which is reactive. getMyUsage is kind of broken and I think I'm going to remove it.